frontend-ssr/services/ExerciseSubmissions.service.ts

```diff
@@ -5,7 +5,8 @@ import HttpService from './Http.service';
 
 class ExerciseService {
   static getSubmissions(page = 0, query = '') : Promise<Submission[]> {
-    return HttpService.get(`${process.env.ENDPOINT}/exercises/submissions?page=${page}&query=${query}`) as unknown as Promise<Submission[]>;
+    return HttpService.get(`${process.env.ENDPOINT}/exercises/submissions?page=${page}&query=${query}`)
+      .then((response) => response) as unknown as Promise<Submission[]>;
```

Comment:
Not sure what you wanted to do here but you added a `.then` which just returns the `response` which is basically useless... You should have called `..json` on the response and return that.